shell/browser/native_window_views.cc

```diff
@@ -904,7 +904,7 @@ bool NativeWindowViews::IsKiosk() {
 }
 
 SkColor NativeWindowViews::GetBackgroundColor() {
-  return root_view_->GetBackground;
+  return root_view_->get_color();
```

Comment:
`RootView` inherits from View: so this'll want to be `background()`

See: https://cs.chromium.org/chromium/src/ui/views/view.h?sq=package:chromium&g=0&l=854-855